NpmAddDependencies.js

```diff
@@ -14,11 +14,33 @@ class NpmAddDependencies {
     process.argv.forEach((val, index) => {
       if (val && index !== 0 && index !== 1) {
         switch (val) {
+          case '--prod':
+          case '--save-prod':
+          case '-P': {
```

Comment:
Is there `-P` standing for `--prod` in `npm`? I am not sure. It stands for `--peer` in yarn.

Reply:
@arfeo I guess so. Weird that `yarn` chose it to mean something different.
> -P, --save-prod: Package will appear in your dependencies. This is the default unless -D or -O are present.
https://docs.npmjs.com/cli/install

Reply:
I think it's unnecessary since 1) we have the `default` switch section; 2) there's a line in the docs: 

> If no `target` argument passed, dependencies are written to `dependencies`.

Reply:
@arfeo Fair enough. Removed the prod flags.